### PR TITLE
Gui: remove Py2 code from several src/Gui .cpp files

### PR DESCRIPTION
**Pull request by luzpaz**
`ApplicationPy.cpp`, `CommandPyImp.cpp`, `OnlineDocumentation.cpp`, `PythonConsole.cpp`, `PythonConsolePy.cpp`, `PythonDebugger.cpp`

### CODE REVIEW

**Comment by luzpaz** [comment]
wmayer, these looked pretty straight-forward, so I went ahead and removed them. Hope that's OK.

File: src/Gui/Selection.cpp [hunk 1]
```diff
@@ -2067,11 +2067,7 @@ PyObject *SelectionSingleton::sCountObjectsOfType(PyObject * /*self*/, PyObject
         return NULL;
 
     unsigned int count = Selection().countObjectsOfType(objecttype, document, resolve);
-#if PY_MAJOR_VERSION < 3
```
Comment (chennes) [inline src/Gui/Selection.cpp:2070]: This is backwards, it's the other branch that needs to be kept (the condition is the opposite of the usual >=3).

Comment (luzpaz) [reply src/Gui/Selection.cpp:2070]: Fixed in e26f7e42c8b80eb5ae30049ec18b2aa4d257200f

File: src/Gui/Selection.cpp [hunk 2]
```diff
@@ -2340,15 +2336,9 @@ PyObject *SelectionSingleton::sSetVisible(PyObject * /*self*/, PyObject *args)
         if(visible == Py_None) {
             vis = -1;
         }
-#if PY_MAJOR_VERSION < 3
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply src/Gui/Selection.cpp:2343]: Fixed in e26f7e42c8b80eb5ae30049ec18b2aa4d257200f

**Comment by luzpaz** [comment]
Ok, perhaps that enough for 1 PR ATM?

**Comment by wwmayer** [comment]
Merged as 6a7184139d

**Pull request closed without merging**